src/modules/esl-event-listener/core/listener.ts

```diff
@@ -91,11 +91,12 @@ export class ESLEventListener implements ESLListenerDefinition, EventListenerObj
     return false;
   }
 
-  /** Handles caught event (used as callback for low-level subscriptions) */
-  public handleEvent(e: Event): void {
-    if (!this.isDelegatedTarget(e)) return;
-    this.handler.call(this.host, e, this);
-    if (this.once) this.unsubscribe();
+  /** Returns exact bound handler (used as callback for low-level subscriptions) */
+  @memoize()
+  public get handleEvent(): EventListener {
```

Comment:
Mb it's better to name it now like this:
```suggestion
  public get handlerEvent(): EventListener {
```

Reply:
It is not possible because this name is declared in native EventListenerObject interfaces